polyfactory/factories/base.py

```diff
@@ -951,6 +956,16 @@ def process_kwargs(cls, **kwargs: Any) -> dict[str, Any]:
                     )
                     continue
 
+                if isinstance(field_value, FieldMeta):
+                    if field_value.annotation is not Null:
+                        field_meta.annotation = field_value.annotation
+                        field_meta.children = field_value.children
+
+                    if field_value.constraints:
+                        if field_meta.constraints is None:
+                            field_meta.constraints = {}
+                        field_meta.constraints.update(field_value.constraints)
+
```

Comment:
Couldn't this just be `cls.get_field_value(field_value, ...)` here instead? The modifying of the `field_meta` directly may have unforeseen consequences if they're cached or something.